spec/support/sufia_factories.rb

```diff
@@ -1,11 +1,23 @@
-def make_generic_file(user, args = {})
+def make_generic_file(user, args={})
+  gf = build_generic_file(user, args)
+  gf.save!
+  gf
+end
+
+def make_generic_file_with_content(user, args={})
```

Comment:
I was running into two issues while testing so I ended up not using this, but this function will generate a GenericFile that has content attached to it. Because it has content it will also have versions.

Here are the two issues I'm running into:

1. Adding content to the let variable generic_file causes to_json to have a content path which will not be consistent across environments.
2. For some reason generating a second instance of InvenioRdmRecordConverter that uses a file with content causes duplicate constant errors which I think are in turn causing more errors. It was a really noisy error message, but the root of it seemed to be with constants being defined twice.

Reply:
Thanks for keeping this in the commit. I ended up using it very minimally, and as long as there is some version, the `let(:json)` object is good enough for me.